deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -142,7 +142,7 @@ cephClusterSpec:
   mon:
     # Set the number of mons to be started. Generally recommended to be 3.
     # For highest availability, an odd number of mons should be specified.
-    count: 3
```

Comment:
I known there is a "DO-NOT-Review", but is this monitor count change intentional?